ServerCore/Pages/Events/AddAdminOrAuthor.cshtml

```diff
@@ -7,48 +7,15 @@
     var roleString = Model.addAdmin ? "admin" : "author";
 }
 
-<h2>Add @roleString</h2>
-
 <div>
-    <a asp-page="/Events/Players">Cancel</a>
-</div>
 
-<h4>Select a user to make an @roleString</h4>
+    <h2>Add @roleString</h2>
 
-<table class="table">
-    <thead>
-        <tr>
-            <th>
-                Name
-            </th>
-            <th>
-                Email
-            </th>
-            <th>
-            </th>
-        </tr>
-    </thead>
-    <tbody>
-        @foreach (var item in Model.Users)
-        {
-        <tr>
-            <td>
-                @item.Name
-            </td>
-            <td>
-                @item.Email
-            </td>
-            <td>
-                @if (Model.addAdmin)
-                {
-                    <a asp-page-handler="AddAdmin" asp-route-userId=@item.ID>Add</a>
```

Comment:
This handler is dead code now, yes?

Reply:
Yes -- this whole section is deleted in the diff

Reply:
These appear to be implemented in AddAdminOrAuthor.cshtml.cs lines 54-110, which is **not** in the diff. [The diff does clearly include the deletion of the old code for team members, though.]

Reply:
Actually fixed this time